types/d3-sankey/index.d.ts

```diff
@@ -0,0 +1,267 @@
+// Type definitions for D3JS d3-sankey module 0.4
+// Project: https://github.com/d3/d3-sankey/
+// Definitions by: Tom Wanzek <https://github.com/tomwanzek>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// Last module patch version validated against: 0.4.2
+
+/**
+ * A helper interface as an extension reference for user-provided properties of
+ * nodes and links in the graph, which are not required or calculated by
+ * the Sankey Layout Generator
+ */
+export interface SankeyExtraProperties { [key: string]: any; }
```

Comment:
Fair enough. I've played with it and it doesn't seem like it introduces any accidental issues, so I'm down.